api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java

```diff
@@ -527,4 +527,50 @@ public CohortDefinition getPregnantWomenAndNotDocumentedWithViralLoadResultsDeno
     cd.setCompositionString("pregnant AND NOT routine");
     return cd;
   }
+
+  /**
+   * Get patients who are breastfeeding
+   *
+   * @return CohortDefinition
+   */
+  public CohortDefinition getPatientsWhoAreBreastfeeding() {
+    CalculationCohortDefinition cd =
+        new CalculationCohortDefinition(
+            "breastfeeding",
+            Context.getRegisteredComponents(BreastfeedingCalculation.class).get(0));
+    cd.addParameter(new Parameter("onDate", "On Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+    return cd;
+  }
+
+  public CohortDefinition getPatientsWhoArePregnant() {
```

Comment:
Can we please rename this function to be more descriptive of what it is doing because right now this is too similar to the function below: `getPatientsWhoArePregnantCohort`

Maybe something like `getPatientsWhoArePregnantAndNotBreastfeeding`

Reply:
ok @nthfloor , will rename to the name you suggested

Reply:
done